portal-frontend/src/app/features/edit-application/edit-application.component.html

```diff
@@ -69,12 +69,19 @@ <h6 *ngIf="application">Application ID: {{ application.fileNumber }} | {{ applic
   </cdk-step>
   <cdk-step label="Proposal">
     <ng-container *ngTemplateOutlet="header"></ng-container>
-    <div *ngIf="cdkStepper.selectedIndex === editAppSteps.Proposal">
+    <div *ngIf="application && cdkStepper.selectedIndex === editAppSteps.Proposal">
```

Comment:
Is an application check required only because of async data retrieval or is it possible not to have an app at all?

Reply:
Just due to async data retrieval, I figured it was cleaner for the components to require a defined application and to use ngIfs higher up.